README.md

```diff
@@ -190,13 +190,18 @@ try {
         String rememberMeId = activityDetails.getRememberMeId();
         String base64Selfie = activityDetails.getBase64Selfie();
         Attribute<Image> selfie = profile.getSelfie();
-        Image selfieValue = selfie.getValue();
+        if (selfie != null) {
+            Image selfieValue = selfie.getValue();
+        }
         Attribute<String> fullName = profile.getFullName();
         Attribute<String> givenNames = profile.getGivenNames();
         Attribute<String> familyName = profile.getFamilyName();
```

Comment:
In the other SDKs we show how to return the values of each attribute here, rather than the Attribute object.

I like returning the attributes here in the way that you've done, as then there's not the potential to error when accessing the value if the attribute isn't present. Should we change the other SDKs to do it in this manner?